content/v2.1/zh-CN/developer/s2i-template.md

```diff
@@ -243,6 +236,8 @@ $ docker run -p 8080:8080  sample-app
 
 ## 第五步 推送镜像并在 KubeSphere 添加 S2i 模版
 
+S2I 模版用于模版定义了应用程序构建的基础环境，包括构建器镜像 (Builder Image) 和 运行时镜像 (Runtime Image)，以及环境参数、描述信息等。
+
 当我们在本地完成 S2I 构建器镜像的测试之后，就可以推送镜像到自定义的镜像仓库当中，并创建构建器模版 `yaml` 文件：
```

Comment:
prefer add docs to explain s2ibuildertemplate fields 
https://github.com/kubesphere/docs.kubesphere.io/pull/594/files#diff-2ecf9ffc61f39d9147e83d8a937b9120R18

Reply:
ref https://kubesphere.com.cn/forum/d/728-b2i/9